manifests/init.pp

```diff
@@ -88,6 +88,7 @@
   $lingering_timeout                                         = '5s',
   Optional[Enum['on', 'off']] $etag                          = undef,
   Optional[String] $events_use                               = undef,
+  Optional[Array[String]] $debug_connection                  = undef,
```

Comment:
How are these specified? Can you use `Stdlib::Host` instead of `String? If not, `String[1]` is probably better than allowing an empty string.

Reply:
Thought to keep Array or Sting, but let's stick with array and cleaner code.
Done.